### DISCUSSION TITLE
On pypi.org, add project link for Changelog

### DISCUSSION THREAD

**Discussion by vlcinsky** [sqlalchemy/alembic#1306] [Usage Questions] [answered]
pypi.org provides handy feature of project links, one type may lead to changelog.

Alembic currently does not use this type of project link.

Adding changelog project link would simplify review of latest changes of the project directly from pypi.org, link would lead probably to https://alembic.sqlalchemy.org/en/latest/changelog.html (without specified version).

**Comment by CaselIT** [comment, marked as answer]
Hi,

Sure, PR are welcome. I think that link just needs to be added to the setup.cfg file

**Comment by vlcinsky** [comment]
See PR #1309
